<ui version="4.0" >
 <class>eduversumGui</class>
 <widget class="QWidget" name="eduversumGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Eduversum Installer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <item row="4" column="0" >
    <widget class="QFrame" name="frame2" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="2" >
       <spacer name="horizontalSpacer_4" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>50</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" >
       <widget class="QPushButton" name="aboutPushButton" >
        <property name="text" >
         <string>about</string>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QPushButton" name="nextPushButton" >
        <property name="text" >
         <string>next </string>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="titleLabel" >
     <property name="text" >
      <string>Welcome</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QStackedWidget" name="contentStackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="welcomePage" >
      <layout class="QGridLayout" name="gridLayout_10" >
       <property name="margin" >
        <number>2</number>
       </property>
       <item row="0" column="1" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>141</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>161</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="QFrame" name="frame" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth" >
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout_5" >
          <item row="0" column="0" >
           <widget class="QRadioButton" name="radioButton" >
            <property name="text" >
             <string>Res&amp;ynchronize the package index files (apt-get update)</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QRadioButton" name="radioButton2" >
            <property name="text" >
             <string>Skip Resynchroni&amp;zation</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2" >
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>160</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>140</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="processPage" >
      <layout class="QGridLayout" name="gridLayout_6" >
       <property name="margin" >
        <number>2</number>
       </property>
       <item row="0" column="0" >
        <widget class="QFrame" name="progressFrame" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth" >
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item row="0" column="0" >
           <widget class="QStackedWidget" name="progressStackedWidget" >
            <property name="maximumSize" >
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="contextMenuPolicy" >
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="lineWidth" >
             <number>0</number>
            </property>
            <property name="currentIndex" >
             <number>1</number>
            </property>
            <widget class="QWidget" name="progressPage1" >
             <layout class="QGridLayout" name="gridLayout_7" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>0</number>
              </property>
              <item row="0" column="0" >
               <widget class="QProgressBar" name="progressBar" >
                <property name="minimum" >
                 <number>0</number>
                </property>
                <property name="value" >
                 <number>24</number>
                </property>
                <property name="textVisible" >
                 <bool>true</bool>
                </property>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance" >
                 <bool>false</bool>
                </property>
                <property name="textDirection" >
                 <enum>QProgressBar::TopToBottom</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="progressPage2" >
             <layout class="QGridLayout" name="gridLayout_8" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>0</number>
              </property>
              <item row="0" column="0" >
               <widget class="QFrame" name="progressBarFrame" >
                <property name="minimumSize" >
                 <size>
                  <width>0</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>16777215</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="frameShape" >
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow" >
                 <enum>QFrame::Raised</enum>
                </property>
                <widget class="QFrame" name="pFrame" >
                 <property name="geometry" >
                  <rect>
                   <x>2</x>
                   <y>2</y>
                   <width>111</width>
                   <height>24</height>
                  </rect>
                 </property>
                 <property name="minimumSize" >
                  <size>
                   <width>0</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>16777215</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="palette" >
                  <palette>
                   <active>
                    <colorrole role="Base" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>255</red>
                       <green>255</green>
                       <blue>255</blue>
                      </color>
                     </brush>
                    </colorrole>
                    <colorrole role="Window" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>85</red>
                       <green>170</green>
                       <blue>255</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </active>
                   <inactive>
                    <colorrole role="Base" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>255</red>
                       <green>255</green>
                       <blue>255</blue>
                      </color>
                     </brush>
                    </colorrole>
                    <colorrole role="Window" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>85</red>
                       <green>170</green>
                       <blue>255</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </inactive>
                   <disabled>
                    <colorrole role="Base" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>85</red>
                       <green>170</green>
                       <blue>255</blue>
                      </color>
                     </brush>
                    </colorrole>
                    <colorrole role="Window" >
                     <brush brushstyle="SolidPattern" >
                      <color alpha="255" >
                       <red>85</red>
                       <green>170</green>
                       <blue>255</blue>
                      </color>
                     </brush>
                    </colorrole>
                   </disabled>
                  </palette>
                 </property>
                 <property name="autoFillBackground" >
                  <bool>true</bool>
                 </property>
                 <property name="frameShape" >
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow" >
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="detailsPushButton" >
            <property name="text" >
             <string>show details</string>
            </property>
            <property name="checkable" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QTextBrowser" name="outputTextBrowser" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="packagePage" >
      <layout class="QGridLayout" name="gridLayout_3" >
       <property name="margin" >
        <number>2</number>
       </property>
       <item row="2" column="0" >
        <widget class="QTreeWidget" name="treeWidget" >
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="headerHidden" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QTextBrowser" name="descriptionTextBrowser" />
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="warningLabel" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="Base" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window" >
             <brush brushstyle="SolidPattern" >
              <color alpha="127" >
               <red>241</red>
               <green>76</green>
               <blue>76</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window" >
             <brush brushstyle="SolidPattern" >
              <color alpha="127" >
               <red>241</red>
               <green>76</green>
               <blue>76</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base" >
             <brush brushstyle="SolidPattern" >
              <color alpha="127" >
               <red>241</red>
               <green>76</green>
               <blue>76</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window" >
             <brush brushstyle="SolidPattern" >
              <color alpha="127" >
               <red>241</red>
               <green>76</green>
               <blue>76</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="autoFillBackground" >
          <bool>true</bool>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:11pt; font-weight:400; font-style:normal;">
&lt;table>&lt;tr>&lt;td>&lt;img src="/home/wuertz/qhttp/icons/warning.png">&lt;/td>&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Warning: Because of dependencies there will be more packages (red marked) removed, then you have chosen.&lt;/p>&lt;/td>&lt;/tr>&lt;/table>&lt;/body>&lt;/html></string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="noteLabel" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="Base" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window" >
             <brush brushstyle="SolidPattern" >
              <color alpha="127" >
               <red>250</red>
               <green>164</green>
               <blue>49</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window" >
             <brush brushstyle="SolidPattern" >
              <color alpha="127" >
               <red>250</red>
               <green>164</green>
               <blue>49</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base" >
             <brush brushstyle="SolidPattern" >
              <color alpha="127" >
               <red>250</red>
               <green>164</green>
               <blue>49</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window" >
             <brush brushstyle="SolidPattern" >
              <color alpha="127" >
               <red>250</red>
               <green>164</green>
               <blue>49</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="autoFillBackground" >
          <bool>true</bool>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Arial'; font-size:11pt; font-weight:400; font-style:normal;">
&lt;table border="0" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px;" cellspacing="2" cellpadding="0">
&lt;tr>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;img src="/home/wuertz/qhttp/icons/hint.png" />&lt;/p>&lt;/td>
&lt;td>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Note: Because of dependencies there will be more packages (orange marked) installed/updated, then you have chosen.&lt;/p>&lt;/td>&lt;/tr>&lt;/table>&lt;/body>&lt;/html></string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="downloadPage" >
      <layout class="QGridLayout" name="gridLayout_9" >
       <item row="0" column="0" >
        <widget class="QLabel" name="downloadLabel" >
         <property name="text" >
          <string>Download</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QProgressBar" name="downloadProgressBar" >
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QTreeWidget" name="downloadTreeWidget" >
         <property name="iconSize" >
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Version</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Source</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite_2" />
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
